CONTRIBUTING.md

```diff
@@ -80,7 +80,7 @@ If your PR contains large changes, e.g. component refactor or new components, pl
 
 ### Code review guidance
 
-Committers will rotate reviewing the code to make sure all the PR will be reviewed timely and by at least one committer before merge. If we aren't doing our job (sometimes we drop things). And as always, we welcome volunteers for code review.
```

Comment:
Code review is assigned to Committer, not Committee.

Reply:
Whether the text under the "Now how about try become a committer?" heading is modified correctly

Reply:
If correct I will roll back what you marked

Reply:
I think the word 'Committer' is fine. No 'Committee' is needed.

Reply:
Ok, I'll just fix the link failure error.